aws/resource_aws_kinesis_firehose_delivery_stream.go

```diff
@@ -1275,6 +1291,39 @@ func resourceAwsKinesisFirehoseDeliveryStream() *schema.Resource {
 							},
 						},
 
+						"vpc_config": {
```

Comment:
@rajholla Could you rename this to `vpc_configuration` to match the AWS API? Thanks.

Reply:
@ewbankkit `vpc_config` is consistent with other resources like [eks](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/eks_cluster) and [lambda](https://registry.terraform.io/providers/hashicorp/aws/latest/docs/resources/lambda_function) 
Is there any advantage changing this ?

Reply:
For Lambda the AWS API names the field `VpcConfig` but for EKS it's named `ResourceVpcConfig` in the API so we are not consistently sticking to the API naming 😄.
Staying with `vpc_config` is fine.